core/tasks.py

```diff
@@ -12,40 +13,49 @@
 from chronam.core.models import Batch, OcrDump
 from chronam.core.batch_loader import BatchLoader
 
-logger = logging.getLogger(__name__)
+_logger = logging.getLogger(__name__)
 
+@task
+def load_page_in_cache(url):
+    try:
+        _logger.debug("Requesting page [%s]" % url)
+        r = requests.get(url)
+        _logger.debug("status %s" % r.status_code)
+        r.raise_for_status() #throw an error if status is not OK
```

Comment:
Should we be using exceptions at all for this or simply having a `if not requests.ok:` call? Given that this is cache warming I'm not sure we want to enable Celery's retries but perhaps we might want some logging to confirm that e.g. a typo in the requested URL wouldn't mean that we just warmed a bunch of redirects due to a missing trailing slash.